profile/README.md

```diff
@@ -0,0 +1,138 @@
+<div align="center">
+    <img src="https://github.com/E-constellatio/.github/blob/main/profile/resources/E-constellatio.svg" alt="Logo" height="150px">
+    <h1>
+        Welcome in E-constellatio Github organization
+    <img src="https://media.giphy.com/media/hvRJCLFzcasrR4ia7z/giphy.gif" alt="Hi hand" width="30px"/>
+    </h1>
+</div>
+
+This organization contain repositories of E-constellatio website. Repositories of other products can be found
+in [Libra foundation organization](https://github.com/Libra-foundation)
+and [Correlatio organization](https://github.com/Correlatio-company).
+
+<h2 align="center">
+    What is E-constellatio ?
+</h2>
+
+<p>
+    E-constellatio is a French and young company based in Belfort created by 3 friends. Company is mainly specialized in digital solutions, and is divided in 3 branches with their field of activity.
+</p>
+
+<h2 align="center">
+    Branches
+</h2>
+
+<div align="center">
+    <img src="https://github.com/E-constellatio/.github/blob/main/profile/resources/LuxDigitalis.svg" alt="Lux digitalis logo" height="50"/>
+    <details>
+        <summary>
+            <h3>
+                Lux Digitalis
+            </h3>
+        </summary>
+        <div align="left">
+            <p>
+                Lux Digitalis is the E-constellatio branch which offer digital consulting and freelancing in web design, integration (front end and back end), AI and cloud infrastructure. Its strong experience in web development is showing through products offered by other E-constellatio branches. We provide skilled people who mastered in a plenty of technologies such as : 
+            </p>
+        </div>
+        <a href="https://www.typescriptlang.org/">
```

Comment:
We might want to do a map of our competencies to better dispatch the work, and ensure we didn't forget any of our strong points.

Reply:
Yes, I totally agree, I will plane a summit with all employees.